ToolsAcct/code-pipeline.yaml

```diff
@@ -107,20 +107,16 @@ Resources:
       Source:
         Type: CODEPIPELINE
         BuildSpec: |
-          version: 0.1
+          version: 0.2
+
           phases:
             install:
               commands:
-                - printenv
-                - ls -R
-                - pip install -r requirements.txt -t "$PWD"
+                - gem install cfn-nag
             build:
               commands:
-                - aws cloudformation package --template-file lambda-cloudformation.yaml --s3-bucket $S3Bucket --s3-prefix sample-lambda/codebuild --output-template-file samtemplate.yaml
-          artifacts:
-            files: samtemplate.yaml
-            discard-paths: yes
-
+                #- aws cloudformation validate-template --template-body file://$(pwd)/template.yaml
+                - cfn_nag_scan --input-path ./template.yaml 
```

Comment:
Thanks for taking time on enhancements. I like the idea of using cfn-nag, however, this PR seems to be removing cloudformation package, which is an essential must have step